ChangeLog.md

```diff
@@ -1,4 +1,12 @@
-﻿## 6.1.0 - May 2018
+﻿## 6.1.1 - May 2018
+#### AzureRM.Resources
+* Revert change to `New-AzureRmADServicePrincipal` that gave service principals `Contributor` permissions over the current subscription if no values were provided for the `Role` or `Scope` parameters
+    - If no values are provided for `Role` or `Scope`, the service principal is created with no permissions
+    - If a `Role` is provided, but no `Scope`, the service principal is created with the specified `Role` permissions over the current subscription
+    - If a `Scope` is provided, but no `Scope`, the service principal is created with `Contributor` permissions over the specified `Scope`
```

Comment:
Role?

Reply:
@darshanhs90 oops, thanks. Fixed this in all occurrences.